functions/utils/database.js

```diff
@@ -1,7 +1,7 @@
 const { db } = require("./admin");
 
-const addDocument = (collection, document) => {
-  db.collection(collection).add(document);
+const addDocument = (collection, id, document) => {
+  return db.collection(collection).doc(id).set(document);
 };
```

Comment:
This is setting a document versus adding a document, while similar they are different. I would keep both of these functions but modify addDocument to return the response. Your function will be really useful for application so definitely keep it!